Dockerfile

```diff
@@ -31,6 +31,7 @@ COPY --from=redis /usr/local/ /usr/local/
 ADD . /build
 
 RUN ./deps/readies/bin/getpy2
+RUN test -f /usr/bin/python || ln -s /usr/bin/python2 /usr/bin/python
```

Comment:
Also possible: `FIX=1 ./deps/readies/bin/getpy2`

Reply:
We're not merging this back to master. It's a cut-throat patch. Didn't know, thanks - will use next time!